### PR TITLE
Embed seaweedfs into local run.

### PR DESCRIPTION
**Pull request by tjholm**
Using seaweedfs over minio due to licensing.

### CODE REVIEW

File: pkg/run/seaweed.go [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+package run
+
+import (
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/pkg/errors"
+
+	// "github.com/nitrictech/minio/cmd"
```
Comment (asalkeld) [inline pkg/run/seaweed.go]: remove this if you update the PR

File: pkg/run/seaweed.go [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+	apiPort int // external API port from the seaweedfs s3 API gateway
+}
+
+const (
+	bucketsDir = "./.nitric/buckets"
+	logsDir    = "/.nitric/logs"
+	runPerm    = os.ModePerm // NOTE: octal notation is important here!!!
+)
+
+// Start - Start the local Minio server
```
Comment (asalkeld) [inline pkg/run/seaweed.go]: nit: there is a lot of name confusion between minio and seaweed.. could just s/minio/seaweed?

File: pkg/run/storage.go [hunk 3]
```diff
@@ -37,8 +37,8 @@ type RunStorageService struct {
 
 const (
```
Comment (asalkeld) [inline pkg/run/storage.go]: SEAWEED_ ...

Comment (tjholm) [reply pkg/run/storage.go]: These are actually no longer used, so will just delete them (the client still provides them but the server doesn't care.

File: pkg/run/seaweed.go (hunk 1 above)
Comment (tjholm) [reply pkg/run/seaweed.go]: removed

File: pkg/run/storage.go (hunk 3 above)
Comment (tjholm) [reply pkg/run/storage.go]: removed

File: pkg/run/seaweed.go (hunk 2 above)
Comment (tjholm) [reply pkg/run/seaweed.go]: fixed

**Review by asalkeld — APPROVED** [review]

**Pull request merged**